src/frontend/screens/Login/index.tsx

```diff
@@ -4,13 +4,13 @@ import Runner from './components/Runner'
 import { useTranslation } from 'react-i18next'
 import { useNavigate } from 'react-router'
 
-import { ReactComponent as EpicLogo } from 'src/assets/epic-logo.svg'
-import { ReactComponent as GOGLogo } from 'src/assets/gog-logo.svg'
+import { ReactComponent as EpicLogo } from 'frontend/assets/epic-logo.svg'
+import { ReactComponent as GOGLogo } from 'frontend/assets/gog-logo.svg'
 
-import { LanguageSelector, UpdateComponent } from 'src/components/UI'
-import { FlagPosition } from 'src/components/UI/LanguageSelector'
+import { LanguageSelector, UpdateComponent } from '../../components/UI'
```

Comment:
just wondering, can't these be `frontend/...` like the others?

Reply:
It absolutely can (and I have no doubt you'll find more of these elsewhere)

Note that the same thing doesn't apply for the backend, for some reason trying to do `import ... from 'backend/someFile'` gives an error that it can't resolve `backend`. *I think* it's caused by some module we have exporting a `Backend` object, although I'm really not sure there.